homeassistant/components/googlehome/sensor.py

```diff
@@ -35,11 +34,15 @@
     if discovery_info is None:
         _LOGGER.warning(
             "To use this you need to configure the 'googlehome' component")
+        return
 
     devices = []
     for condition in SENSOR_TYPES:
+        await hass.data[CLIENT].update_info(discovery_info['host'])
```

Comment:
We don't need to do this in the loop. Move it up outside the loop.

Reply:
Roger that 👍